ecommerce-ios/Extensions/View+NavigationBar.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  View+NavigationBar.swift
+//  ecommerce-ios
+//
+//  Created by Su T. Nguyen on 13/05/2021.
+//
+
+import SwiftUI
+
+public extension View {
+
+    func navigationBarLargeTitleItems<T>(trailing: T) -> some View where T: View {
+        overlay(NavigationBarLargeTitleItems(trailing: trailing)
+                    .frame(width: 0, height: 0))
+    }
+}
+
+private struct NavigationBarLargeTitleItems<T: View>: UIViewControllerRepresentable {
+
+    typealias UIViewControllerType = Wrapper
+
+    private let trailingItems: T
+
+    init(trailing: T) {
+        self.trailingItems = trailing
+    }
+
+    func makeUIViewController(context: Context) -> Wrapper {
+        Wrapper(representable: self)
+    }
+
+    func updateUIViewController(_ uiViewController: Wrapper, context: Context) { }
+
+    class Wrapper: UIViewController {
+
+        private let representable: NavigationBarLargeTitleItems?
+
+        init(representable: NavigationBarLargeTitleItems) {
+            self.representable = representable
+            super.init(nibName: nil, bundle: nil)
+        }
+
+        required init?(coder: NSCoder) {
+            self.representable = nil
+            super.init(coder: coder)
+        }
+
+        override func viewWillAppear(_ animated: Bool) {
+            guard let representable = self.representable else { return }
+            guard let navigationBar = self.navigationController?.navigationBar else { return }
+            guard let navigationBarLargeTitleView = NSClassFromString("_UINavigationBarLargeTitleView")
```

Comment:
Because the type is `class`, I prefer to name it with suffix `Class`. What do you think? 

```suggestion
            guard let LargeTitleViewClass = NSClassFromString("_UINavigationBarLargeTitleView")
```